src/sp_utils.c

```diff
@@ -50,14 +50,22 @@ void sp_log_msg(char const* restrict feature, int type,
   va_end(args);
 
   const char* client_ip = get_ipaddr();
+  const char* logtype = "";
+  if (type == SP_LOG_SIMULATION) {
```

Comment:
`SP_LOG_SIMULATION` is defined as a `E_WARNING`

Reply:
Yes, but i changed the value on purpose. see below